<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>471</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="principal">
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>52</y>
        <width>391</width>
        <height>421</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pesquiar_colaborador">
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>341</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Pesquisar</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>441</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cadastro_colaborador">
     <widget class="QLineEdit" name="edt_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_nome_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_cpf">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Códico:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Nome Completo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Função:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> CPF:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_funcao">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> RG:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_rg">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Endereço:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_endereco">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>331</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Nº</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_numeroEnd">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Bairro:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_bairro">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Cidade:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> UF</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_cidade">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_uf">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>230</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> CNH:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_cnh">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_salvar">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>430</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_fechar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>430</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Fechar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cadastro_usuario">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>381</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="edt_nome">
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>3</height>
          </size>
         </property>
         <property name="placeholderText">
          <string> Digite Nome Completo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_login">
         <property name="placeholderText">
          <string>Digite o Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_senha">
         <property name="placeholderText">
          <string>Digite a Senha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_c_senha">
         <property name="placeholderText">
          <string> Repita a Senha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_salvar_user">
         <property name="text">
          <string>Salvar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_fechar_2">
         <property name="text">
          <string>Fechar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="configurar_logo"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>474</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCadastro">
    <property name="title">
     <string>Cadastro</string>
    </property>
    <widget class="QMenu" name="menuColaborador">
     <property name="title">
      <string>Colaborador</string>
     </property>
     <addaction name="actionPesquisar"/>
     <addaction name="actionCadastrar"/>
    </widget>
    <addaction name="menuColaborador"/>
    <addaction name="actionUser"/>
   </widget>
   <widget class="QMenu" name="menuRelatorio">
    <property name="title">
     <string>Relatorio</string>
    </property>
    <addaction name="actionRelat_rio_por_Colaborador"/>
   </widget>
   <widget class="QMenu" name="menuConfirga_o">
    <property name="title">
     <string>Confirgação</string>
    </property>
    <addaction name="actionTrocar_Logo"/>
   </widget>
   <addaction name="menuCadastro"/>
   <addaction name="menuRelatorio"/>
   <addaction name="menuConfirga_o"/>
  </widget>
  <action name="actionUser">
   <property name="text">
    <string>Usuário</string>
   </property>
  </action>
  <action name="actionRelat_rio_por_Colaborador">
   <property name="text">
    <string>Relatório por Colaborador</string>
   </property>
  </action>
  <action name="actionPesquisar">
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </action>
  <action name="actionCadastrar">
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </action>
  <action name="actionTrocar_Logo">
   <property name="text">
    <string>Trocar Logo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
